### PR TITLE
Added React Best Practices Post

### PR DESCRIPTION
**Pull request by HafizMBilal009**

### CODE REVIEW

File: _posts/2021-03-10-react-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+const working_hours = 10; //bad approach
+
+const workingHours = 10; //good approach
+
+const get_sum = (a, b) => a + b; // bad approach
+
+const getSum = (a, b) => a + b; //good approach
+```
+
+## Id & Class Names Should be in kebab-case
```
Comment (fahadsiddiqui) [inline _posts/2021-03-10-react-best-practices.md:137]:
```suggestion
## ID & Class Names Should be in kebab-case
```

File: _posts/2021-03-10-react-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+  return 3;
+}
+
+//Good approach
+name === "Ali" ? 1 : name === "Bilal" ? 2 : 3;
+```
+
+## Make `index.js` File Name to Minimize Importing Complexity
+
+If you have a file named `index.js` in a directory named `actions` and you want to import action from it in your component, your import would be like this:
```
Comment (fahadsiddiqui) [inline _posts/2021-03-10-react-best-practices.md:284]:
```suggestion
If you have a file named `index.js` in a directory named `actions` and you want to import action from it in your component, your import would be like this
```

Comment (fahadsiddiqui) [reply _posts/2021-03-10-react-best-practices.md:284]: Remove trailing `:` elsewhere.

**Review by fahadsiddiqui — CHANGES_REQUESTED** [review]

File: _posts/2021-03-10-react-best-practices.md [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+While working on a React App, try to follow these coding practices to have a better development experience:
+
+## Use Visual Studio Code as Your IDE
+
+Visual Studio Code has several features that a React developer loves. VS Code gives a lot of useful extensions to make the development environment better. For React, here some useful extensions which will assist you during development:
+
+- Prettier
+- ES Lint
+- JavaScript (ES6) code snippets
+- Reactjs code snippets
```
[the remark this answers is not in the document]
Comment (HafizMBilal009) [reply _posts/2021-03-10-react-best-practices.md:18]: This is the exact name of the extension

**Pull request merged**